### DISCUSSION TITLE
运行失败

### DISCUSSION THREAD

**Discussion by SunLang115** [RapidAI/RapidRAG#6] [Q&A]
AttributeError: module 'streamlit' has no attribute 'toast'
上传文档加载知识库会报这个错

**Comment by SunLang115** [comment]
> AttributeError: module 'streamlit' has no attribute 'toast' 上传文档加载知识库会报这个错误

streamlit需要1.25.0才可以 代码里的1.24.0不支持toast了

**Comment by SWHL** [comment]
感谢指出，是我版本搞错了，已经更改。

**Comment by SunLang115** [comment]
请问上传文档加载数据库报这个错是因为什么呢？
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xa1 in position 0: invalid start byte
![Uploading 屏幕截图 2023-08-02 223308.png…]()

**Comment by SWHL** [comment]
应该是内容编码问题。你先试试tests目录下的简单文档，是否能用，确定环境没问题之后
再测试自己的文档

**Comment by SunLang115** [comment]
> 应该是内容编码问题。你先尝试tests目录下的简单文档，是否能用，确定环境没问题之后 再测试自己的文档

我试了长安三万里的pdf，还是一样的问题

**Comment by SunLang115** [comment]
> > 应该是内容编码问题。你先尝试tests目录下的简单文档，是否能用，确定环境没问题之后再测试自己的文档
> 
> 我试了长安三万里的pdf，还是一样的问题

基座模型回答没问题，只要涉及到加载向量化都会有问题

**Reply by SWHL** [reply]
需要你这里提供操作系统，python版本，报错截图，我才方便排查问题。
我这里是Windows 10, python 3.10， 运行没有问题
